### PR TITLE
Closes #3278: Adds "image right" layout options to Marketing Cloud export

### LINKED ISSUE #3287: Add layout options for use in Marketing Cloud
## Motivation
I want the option to have the image to the right of the text in my email rather than just image to the left of the text.

## Proposed Resolution
Add new layouts to Quickstart for Marketing Cloud module. 
Add links to the content page to the new layouts.

## Additional context
Row 4: https://arizona.app.box.com/s/t64vuev8tk842tgm4id9ynkin6o3ezpr

### PR DESCRIPTION
**Pull request by danahertzberg**
## Description
Adds "image right" layout options to marketing cloud export.

- [x] Update "how does this work" accordion
- [x] Add "Image left" to original options
- [x] Put image on top of content in mobile

## Related issues
Closes #3287 

## How to test
1. Navigate to Content for email (/admin/content/marketing-cloud)
2. Find a News item
3. On the dropdown for "copy email snippet", select:
   a. 70/30 Layout (Image right)
   b. 50/50 Layout (Image right)
4. Paste the copied link in a new window
   a. Ensure link can be accessed anonymously
   b. Ensure layout is the same but with the image on the right

### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
   - [x] Bug fix
   - [ ] Accessibility, performance, or security improvement
   - [ ] Critical institutional link or brand change
   - [ ] Adding experimental module
   - [x] Update experimental module
- **Minor release changes**
   - [x] New feature
   - [ ] Breaking or visual change to existing behavior
   - [ ] Upgrade experimental module to stable
   - [ ] Enable existing module by default or database update
   - [ ] Non-critical brand change
   - [ ] New internal API or API improvement with backwards compatibility
   - [ ] Risky or disruptive cleanup to comply with coding standards
   - [ ] High-risk or disruptive change (requires upgrade path, risks regression, etc.)
- **Other or unknown**
   - [ ] Other or unknown

### Drupal core
- **Patch release changes**
   - [ ] Security update
   - [ ] Patch level release (non-security bug-fix release)
   - [ ] Patch removal that's no longer necessary
- **Minor release changes**
   - [ ] Major or minor level update
- **Other or unknown**
   - [ ] Other or unknown

### Drupal contrib projects
- **Patch release changes**
   - [ ] Security update
   - [ ] Patch or minor level update
   - [ ] Add new module
   - [ ] Patch removal that's no longer necessary
- **Minor release changes**
   - [ ] Major level update
- **Other or unknown**
   - [ ] Other or unknown

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by its-me-mario** [comment]
On the 70/30 layout
- Update the class name from qs-layout-3070 to: **qs-layout-7030**. 
- On the 70% width <td> change padding-left to: **padding-right**.

On the 50/50 layout
- Change the width attribute from width="200" to: **width="270"**

5050-layout
7030-layout

File: modules/custom/az_marketing_cloud/templates/node--view--az-marketing-cloud--50-50-layout-image-right.html.twig [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+                    </tr>
+                  </tbody>
+                </table>
+              </td>
+              <td valign="top" class="responsive-td" style="width: 50%; padding-left: 7.5px;">
+                <table width="100%" cellspacing="0" cellpadding="0" role="presentation">
+                  <tbody>
+                    <tr>
+                      <td align="center">
+                        <img class="qs-img" src="{{ images.az_marketing_cloud_50_50_layout }}" alt="" width="270" style="display: block; height: auto; width: 100%;">
```
Comment (danahertzberg) [inline modules/custom/az_marketing_cloud/templates/node--view--az-marketing-cloud--50-50-layout-image-right.html.twig:106]: @its-me-mario Changing to `width="270"` makes this inconsistent with the original 50/50 Layout (Image left). Is that okay?

Comment (its-me-mario) [reply modules/custom/az_marketing_cloud/templates/node--view--az-marketing-cloud--50-50-layout-image-right.html.twig:106]: I would say yes.

The [original code snippets](https://codepen.io/itsmemario/pen/GRXNPRL) was using 200 for 30/70 and 270 for 50/50, so it must have been missed.

The image left should really be updated, but the width attribute is a fallback if the style value is broken in some way and I'm not aware of that being a use case for the email clients we support.

Comment (danahertzberg) [reply modules/custom/az_marketing_cloud/templates/node--view--az-marketing-cloud--50-50-layout-image-right.html.twig]: Thank you! I'll make them both 270 in this PR and test it out.

**Review by its-me-mario — APPROVED** [review]

**Comment by danahertzberg** [comment]
✅ Also approved by Kara Kaczmarzyk via Slack

**Review by camikazegreen — APPROVED** [review]

**Review by joeparsons — APPROVED** [review]

**Review by tadean — APPROVED** [review]

**Pull request merged**